gateleen-hook/src/main/resources/gateleen_hooking_schema_hook

```diff
@@ -37,6 +37,13 @@
 		"destination": {
 			"type": "string"
 		},
+        "translateStatus": {
+            "description": "Mapping to transform backend HTTP status.",
+            "type": "object",
+            "additionalProperties": {
+                "type": "number"
```

Comment:
Why floating point number?

Reply:
For uncertain response codes like 301.5 (Redirect semi-permanently).

Reply:
I've never seen that so far. Is there any spec describing this? (So I can learn about them) Only thing I could find was [RFC2616](https://www.ietf.org/rfc/rfc2616.txt). But I cannot see any hint for codes like 301.5. What am I missing?

> 6.1.1 Status Code and Reason Phrase
> 
> The Status-Code element **is a 3-digit integer** result code ...

Reply:
Oops, I forgot the smiley, it was irony :)

Reply:
Good one 🤣. Or like 201.5 Almost Accepted